### ISSUE TITLE
httptools failed to download on python 3.13

### ISSUE THREAD

**Issue by codebased** [MagicStack/httptools#108]
Hi , I am installing on mac m1 with python version 3.13

Caused by: Failed to fetch wheel: httptools==0.6.1
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  113 |     int little_endian, int is_signed, int with_exceptions);
      |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
httptools/parser/parser.c:10136:70: error: too few arguments to function call, expected 6, have 5
 10134 |                 int ret = _PyLong_AsByteArray((PyLongObject *)v,
       |                           ~~~~~~~~~~~~~~~~~~~
 10135 |                                               bytes, sizeof(val),
 10136 |                                               is_little, !is_unsigned);

**Comment by matan1008** [comment]
Welcome to the club! :crying_cat_face: 

You might want to look here https://github.com/MagicStack/httptools/issues/104

**Comment by fantix** [comment]
Fixed in [0.6.2](https://pypi.org/project/httptools/0.6.2/)
